### PR TITLE
Tags support for send_with

### PR DESCRIPTION
**Pull request by echobass**

### CODE REVIEW

File: lib/send_with_us/api.rb [hunk 1]
```diff
@@ -47,9 +47,12 @@ def send_with(email_id, to, data = {}, from = {}, cc = {}, bcc = {}, files = [],
       if version_name
         payload[:version_name] = version_name
       end
-      if headers
+      if headers.any?
```
Comment (demoore) [inline lib/send_with_us/api.rb]: good catch

File: test/lib/send_with_us/api_request_test.rb [hunk 2]
```diff
@@ -103,6 +103,25 @@ def test_send_with_headers
...
+      {name: 'Ruby Unit Test', address: 'matt@example.com'},
+      {name: 'sendwithus', address: 'matt@example.com'},
+      {},
+      [],
+      [],
+      [],
+      '',
+      'v2',
+      {},
+      ['tag1', 'tag2']
```
Comment (gschier) [inline test/lib/send_with_us/api_request_test.rb]: wow, this function has way too many params :cry:. I would not want to use this client.

**Comment by demoore** [comment]
Readme should be updated :smile:

File: test/lib/send_with_us/api_request_test.rb (hunk 2 above)
Comment (demoore) [inline test/lib/send_with_us/api_request_test.rb]: Agreed

**Comment by echobass** [comment]
@demoore Remind me to once it's merged ;)

**Comment by gschier** [comment]
:+1:

**Pull request merged**